integration/go.mod

```diff
@@ -359,3 +359,5 @@ require (
 replace github.com/onflow/flow-go => ../
 
 replace github.com/onflow/flow-go/insecure => ../insecure
+
+replace github.com/onflow/flow-emulator v1.0.0-preview.41.0.20240829134601-0be55d6970b5 => github.com/The-K-R-O-K/flow-emulator v1.0.0-preview.12.0.20240903103527-a68bcda903d1
```

Comment:
please remove this replace, and update to use the commit from your `onflow/flow-emulator` branch
https://github.com/onflow/flow-emulator/pull/756/commits